test/XRP/wallet.test.ts

```diff
@@ -19,7 +19,7 @@ const derivationPathTestCases = {
       '0090802A50AA84EFB6CDB225F17C27616EA94048C179142FECF03F4712A07EA7A4',
     expectedMainNetAddress: 'XVMFQQBMhdouRqhPMuawgBMN1AVFTofPAdRsXG5RkPtUPNQ',
     expectedTestNetAddress: 'TVHLFWLKvbMv1LFzd6FA2Bf9MPpcy4mRto4VFAAxLuNpvdW',
-    messageHex: Buffer.from('test message', 'utf-8').toString('hex'),
+    messageHex: Buffer.from('test message', 'utf8').toString('hex'),
```

Comment:
Totally unrelated, I just like `utf8` better (and when I look at the `Buffer` documentation, it uses `utf8` as the example string, not `utf-8`.